src/extension.ts

```diff
@@ -33,7 +33,7 @@ export function activate(context: vscode.ExtensionContext) {
 
     context.subscriptions.push(vscode.window.registerTreeDataProvider('azureAppService', appServiceDataProvider));
 
-    initCommand(context, 'appService.Refresh', () => appServiceDataProvider.refresh());
+    initCommand(context, 'appService.Refresh', (node?: NodeBase) => node ? appServiceDataProvider.refresh(node) : appServiceDataProvider.refresh());
```

Comment:
Is it necessary to do the null/undefined check here? Since the node parameter of the refresh method is also optional.